modules/tinymce/CHANGELOG.md

```diff
@@ -8,152 +8,153 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- The `table_use_colgroups` option has been set to `true` by default #TINY-8259
-- The default minimum height of editor has been changed to 100px to prevent the UI disappearing while resizing #TINY-6860
+- Renamed the `mceInsertClipboardContent` command `content` property to `html` to better reflect what data is passed #TINY-8310
+- Renamed the `default_link_target` option to `link_default_target` for both `link` and `autolink` plugins #TINY-4603
+- Renamed the `rel_list` option to `link_rel_list` for the `link` plugin #TINY-4603
+- Renamed the `target_list` option to `link_target_list` for the `link` plugin #TINY-4603
+- The default value for the `link_default_protocol` option has been changed to `https` instead of `http` #TINY-7824
+- The default value for the `element_format` option has been changed to `html` #TINY-8263
+- The default value for the `schema` option has been changed to `html5` #TINY-8261
+- The default value for the `table_style_by_css` option has been changed to `true` #TINY-8259
+- The default value for the `table_use_colgroups` option has been changed to `true` #TINY-8259
 
 ### Fixed
 - The object returned from the `editor.fire()` API was incorrect if the editor had been removed #TINY-8018
 - The `editor.selection.getContent()` API did not respect the `no_events` argument #TINY-8018
+- The `editor.annotator.remove` API did not keep selection when removing the annotation #TINY-8195
 - The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API #TINY-8018
+- The `beforeinput` and `input` events would sometimes not fire as expected when deleting content #TINY-8168 #TINY-8329
 - The `table` plugin would sometimes not correctly handle headers in the `tfoot` section #TINY-8104
-- The aria labels for the color picker dialog were not translated #TINY-8381
-- The `editor.annotator.remove` did not keep selection when removing the annotation #TINY-8195
 - The `silver` theme UI was incorrectly rendered before plugins had initialized #TINY-8288
+- The aria labels for the color picker dialog were not translated #TINY-8381
+- Fixed sub-menu items not read by screen readers. Patch contributed by westonkd #TINY-8417
 - Dialog labels and other text-based UI properties did not escape HTML markup #TINY-7524
-- Deleting content would sometimes not fire `beforeinput` and `input` events as expected #TINY-8168 #TINY-8329
 - Anchor elements would render incorrectly when using the `allow_html_in_named_anchor` option #TINY-3799
-- Fixed sub-menu items not read by screen readers. Patch contributed by westonkd #TINY-8417
 - The Home or End keys would move out of a editable element contained within a noneditable element #TINY-8201
 
 ### Removed
-- Removed the jQuery integration #TINY-4518
+- Removed support for Microsoft Internet Explorer 11 #TINY-8194 #TINY-8241
+- Removed support for Word from the `paste` plugin #TINY-7493
```

Comment:
Does this need rewording now that we no longer have a paste plugin?

Reply:
I honestly can not think of a good way to reword this to clarify that

Reply:
Yeah nor can I to be honest. The best I could come up with was something like this... Let's leave it as is then if you don't like the suggestion or would prefer to leave it as is.
```suggestion
- Removed support for Word from the opensource paste functionality #TINY-7493
```

Reply:
I suggest `Microsoft Word` instead of just `Word`

Reply:
That's a good point, yeah it should definitely be the full product name. I'll update my suggestion to include that.

Reply:
I'll go with this, thanks!